platform-cloud/docs/tutorials/retry-strategy.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Manage AWS Spot interruptions in Seqera Platform"
+description: "Managing AWS Spot Interruptions in Seqera Platform."
+date: "16 Jul 2024"
+tags: [aws, spot, platform, fusion, retry]
+---
+
+In AWS Batch environments that use Spot instances, tasks can be interrupted when instances are reclaimed. This is a normal part of how Spot instances operate. The frequency of interruptions can be highly variable, based on factors including the wider demand for AWS services. AWS offers insight into the frequency of Spot reclamations with their **instance-advisor** service, which you can find [here](https://aws.amazon.com/ec2/spot/instance-advisor/).
+
+In Seqera Platform, Spot reclamations may appear through log messages like `Host EC2 (instance i-0282b396e52b4c95d) terminated`. These events often result in non-specific exit codes, such as `143` (indicating `SIGTERM`), or sometimes no exit code at all (`-`), depending on the sequence in which the underlying AWS components are shut down. If you're seeing unexpected task failures with one or more of these features, especially with no obvious application error, it's worth reviewing your Spot configuration and retry strategy.
+
+This guide outlines best practices for mitigating the impact of Spot interruptions and ensuring critical tasks can retry or recover reliably.
+
+## Recommended mitigations
+
+### Use an On-Demand compute environment
+
+For workflows with a significant proportion of long-running processes, the costs of, and mitigations necessary for working with Spot may outweigh the benefits. You may find it simpler and cheaper to run those workloads in On-Demand compute environments.
+
+### Move long-running tasks to On-Demand 
+
+Tasks with long runtimes are particularly vulnerable to Spot termination. In Platform, you can explicitly assign critical or long-duration tasks to On-Demand queues and leave other tasks to run in a Spot queue by default:
+
+```
+process {
+	withName: 'run_bcl2fastq' {
+	     queue = 'TowerForge-MyOnDemandQueue'
+	} 
+}
+```
+
+If you don’t already have one, you may need to create an On-Demand compute environment in Seqera Platform. Once it’s available, open the **Compute Environments** tab in Seqera Platform, and open the relevant compute environment. Scroll down to the **Manual Config Attributes** section to view key configuration details, including queue names. Look for the queue name prefixed with `TowerForge-` if it was created by Forge.
```

Comment:
@pinin4fjords, I updated this based on following the instructions directly in Platform and it's a little unclear whether this is what is being recommended. Can you confirm that the aim is to open the CE and verify the configuration?

Reply:
Sorry, the idea here is to derive the queue name to use in the queue directive.